### PR TITLE
Feat/st 656

### PR DESCRIPTION
**Pull request by muradalamv1**
###  https://tools.hmcts.net/jira/browse/ST-656  ###

Created Record listing pages 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by spotstory — DISMISSED** [review]

**Review by muradalamv1 — COMMENTED** [review]
This branch has conflicts

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/HearingType.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    @JsonProperty("JudicialMediation")
+    JUDICIAL_MEDIATION("Judicial mediation (SEN, DD, CS, PHL only)"),
+
+    @JsonProperty("Final")
+    FINAL("Final"),
+
+    @JsonProperty("Interlocutory")
+    INTERLUCOTORY("Interlocutory");
+
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/HearingType.java:26]: Typo, I think.
INTERLOCUTORY?

**Review by osheaf01 — DISMISSED** [review]
Just one change to a typo, otherwise LGTM

**Review by spotstory — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/HearingType.java (hunk 1 above)
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/model/HearingType.java:26]: Done

**Review by spotstory — APPROVED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

**Pull request merged**